### DISCUSSION TITLE
How to only use local.rules

### DISCUSSION THREAD

**Discussion by jsavary87** [Security-Onion-Solutions/securityonion#2469] [General] [answered]
How would I disable all security onion rules except for local.rules? This is for a lab environment on writing suri rules and I do not want to load all the other rules and waste time. I just want to load/update local.rules to test them.

**Comment by defensivedepth** [comment, marked as answer]
We have a new tool that was released with 2.3.21 that might be of help - `so-suricata-testrule $RuleFile $FullPathToPCAP`

Point it to your custom rule and a pcap (full path) and it will run run Suricata with that custom rule + all.rules against the target PCAP. Make sure to run it on a node that has the sensor role:

![image](https://user-images.githubusercontent.com/954732/103243974-83e14f00-4929-11eb-964c-64ec2f1d3d09.png)

**Reply by mmasino-matc** [reply]
I am doing something similar when developing rules. If you comment out the 'cp' line add the line to remove the old 'all.rules' file in the script it will only run the file containing your test rule and not all the rest. 
![image](https://user-images.githubusercontent.com/77351748/154180760-496f1557-3432-47d7-9460-c0b02ce26775.png)
